app/web/features/connections/friends/AddFriendButton.test.tsx

```diff
@@ -1,16 +1,18 @@
 import { render, screen } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import { Empty } from "google-protobuf/google/protobuf/empty_pb";
+import { CONNECTIONS } from "i18n/namespaces";
+import { t } from "test/utils";
 import React, { useState } from "react";
 import { service } from "service";
 import wrapper from "test/hookWrapper";
 
-import { ADD_FRIEND } from "../constants";
 import AddFriendButton from "./AddFriendButton";
 
 const sendFriendRequestMock = service.api.sendFriendRequest as jest.Mock<
   ReturnType<typeof service.api.sendFriendRequest>
 >;
+const { t } = useTranslation([CONNECTIONS]);
```

Comment:
```suggestion
```
This won't work as we can't use hooks outside of a React component. That's why there's the `t` helper from `test/utils` which you have also imported, so this is unneeded (and probably will cause error for redeclaring the same variable)